### DISCUSSION TITLE
Akai s1100 OS 4.3 boot image

### DISCUSSION THREAD

**Discussion by TwistedGenetics** [ZuluSCSI/ZuluSCSI-firmware#242] [Q&A] [answered]
Hi

How do i get the zulu scsi to boot the akai os 4.3 img first?

Kind regards
Jamie

**Comment by MeTalOneOEight** [comment]
From what I read the Akai first checks the Eeprom, then scans the SCSI bus and finally the floppy.
Read the manual if it starts scanning the SCSI bus at ID 0 or 7, then give that ID to the OS img.

**Reply by TwistedGenetics** [reply, marked as answer]
Hi, I tried that’s but it didn’t work

I have managed to now sort the problem out for anyone else who needs help.

You need to have either the original boot disk (floppy) or the Gotek still plugged in with the Zulu scsi also installed and set up. Then under save, select operating system ( same place where you save your programs), use the data will to scroll to operating system, and save it to hard drive with scsi id 5. Id 5 is the default drive at boot up.

**Reply by MeTalOneOEight** [reply]
Using ID5 for the image did not work?

**Reply by TwistedGenetics** [reply]
No, it came up with unrecognised drive and wanted to format it. I’m presuming it’s looking for a drive then a file on that drive.. much like looking for floppy first and the the image file?

**Reply by MeTalOneOEight** [reply]
Which sector size did you try?
Zulu does not read that from the image and sets a default 2048 if nothing was in the filename.
Try 512 in the filename.

**Reply by TwistedGenetics** [reply]
I just renamed the image to HD5.img

**Reply by MeTalOneOEight** [reply]
Using HD5.img as the filename sets it to 512, seems wrong for your image, maybe CD5.img would have given better results.
Zulu is still missing a good auto detection of the block size.

**Comment by TwistedGenetics** [comment]
It’s ok though as I now have 5 x 512mb usable drives (HD2, HD3 HD4, HD5, HD7) with SCSI id 1 left for scsi card in PC and ID 6 is for the sampler. The Akai operating system saved to the first partition on drive HdD5. Each drive is spilt into 16x 32mb (size of memory installed on sampler) so I only lose 1x32mb for Akai OS.

I Taken a backup of the whole SD card and saved it to 5 other cards which all work. This should keep me going for a while  🤣

I may change the akai eprom chips in the future so I don’t have to worry about this
